programs/zstdcli.c

```diff
@@ -992,11 +992,25 @@ int main(int argCount, const char* argv[])
     if (!followLinks) {
         unsigned u;
         for (u=0, fileNamesNb=0; u<filenameIdx; u++) {
+
+#ifdef _MSC_VER
+
+            if (UTIL_isLink(filenameTable[u])) {
+                DISPLAYLEVEL(2, "Warning : %s is a symbolic link, ignoring\n", filenameTable[u]);
+            } else {
+                filenameTable[fileNamesNb++] = filenameTable[u];
+            }
+
+#else
+
             if (UTIL_isLink(filenameTable[u]) && !UTIL_isFIFO(filenameTable[u])) {
```

Comment:
So are we controlling whether we allow pipes based on the follow links option? Has that flag's documentation been updated? I note that this error message no longer really applies.

Reply:
Yes, we are allowing pipes based on this and I just added a comment